### PR TITLE
Enable & disable multiple apps at once 

### LINKED ISSUE #11274: occ doesn't support enabling/disabling multiple apps at once
As of Nextcloud 14.0.0, `occ` still doesn't support enabling/disabling multiple apps at once:

```
$ ./occ --no-interaction app:enable richdocuments spreed

                                                              
  Too many arguments, expected arguments "command" "app-id".  
                                                              

app:enable [-g|--groups GROUPS] [--] <app-id>

$
```

```
$ ./occ --no-interaction app:disable richdocuments spreed

                                                              
  Too many arguments, expected arguments "command" "app-id".  
                                                              

app:disable <app-id>

$ 
```

Given that usage of `occ` is just slow (~ 0.75-1.25 seconds here), enabling/disabling apps takes a lot of time, especially in containers where e.g. some apps always should be (re-)enabled.

```
$ time ./occ --no-interaction app:enable richdocuments
richdocuments enabled

real	0m1.167s
user	0m0.632s
sys	0m0.407s
$ 
```

```
$ time ./occ --no-interaction app:disable richdocuments
richdocuments disabled

real	0m0.746s
user	0m0.546s
sys	0m0.182s
$ 
```

Yes, you could perform the steps on the database directly, but that likely isn't supported nor future-safe.

### PR DESCRIPTION
**Pull request by kesselb**
Close #11274

Enable & disable multiple apps at once.

### CODE REVIEW

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

File: core/Command/App/Enable.php [hunk 1]
```diff
@@ -94,7 +147,7 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 	 */
 	public function completeOptionValues($optionName, CompletionContext $context) {
 		if ($optionName === 'groups') {
-			return array_map(function(IGroup $group) {
+			return array_map(function (IGroup $group) {
 				return $group->getGID();
 			}, \OC::$server->getGroupManager()->search($context->getCurrentWord()));
```
Comment (nickvergessen) [inline core/Command/App/Enable.php]:
```suggestion
			}, $this->groupManager->search($context->getCurrentWord()));
```

Comment (kesselb) [reply core/Command/App/Enable.php]: How could i miss this :see_no_evil:

**Review by nickvergessen — APPROVED** [review]

**Comment by kesselb** [comment]
@enoch85 @J0WI @kyrofa fyi. Could be useful for your provision scripts :rocket:

**Comment by enoch85** [comment]
Thanks for this @kesselb!

**Pull request merged**